src/particle/__init__.py

```diff
@@ -17,6 +19,11 @@
 from .particle import Particle, SpinType, Parity, Charge, Inv, Status
 from .particle import ParticleNotFound, InvalidParticle
 
+# Direct access to Particle literals
+from .particle import literals
+
+sys.modules["particle.literals"] = literals
```

Comment:
For my own education, why is this line needed?

Reply:
This is needed for `from particle.literals import ...` to work if I do not move the submodule around. Indeed the statement expects `particle.literals` to be a file/module, and it isn't strictly speaking since at `particle.particle.literals` in reality. Indeed I thought I would keep the structure and old functionality. This line is needed at least to the best of my knowledge to make both statements work. Without it `from particle import literals as lp` would work but not `from particle.literals import Lambda_b_0`.

Reply:
Ah, I see, so this is a pretty clever solution. Thanks for the nice explanation.

Reply:
:+1: